azure-cli/ad/Add-GroupToServicePrincipal.ps1

```diff
@@ -0,0 +1,33 @@
+function Add-GroupToServicePrincipal(
+    [Parameter(Mandatory = $true)]
+    [EnvironmentConfig] $environmentConfig,
+  
+    [Parameter(Mandatory = $true)]
+    [NamingConfig] $namingConfig,
```

Comment:
Break line between type and parameter name

Reply:
Done